support/client/lib/mil-sym/cws.js

```diff
@@ -5333,15 +5469,22 @@ define( function(){
                   "tag": "LINE OF CONTACT",
                   "symbolType": "MultiSegmentLine"
             },
+            "TACGRP.C2GM.GNL.LNE.BNDS": {
+                  "symbolID": "G*GPGLB---****X",
+                  "tag": "BOUNDARY LINE",
+                  "symbolType": "MultiSegmentLine",
+                  "validModifiers": [ "uniqueDesignation1", "uniqueDesignation2", "hostile", "echelon" ]
+            },
             "TACGRP.C2GM.GNL.LNE.FLOT": {
                   "symbolID": "G*GPGLF---****X",
                   "tag": "FORWARD LINE OF OWN TROOPS",
-                  "symbolType": "MultiSegmentLine"
+                  "symbolType": "MultiSegmentLine",
+                  "validModifiers": [ "hostile" ]
             },
             "TACGRP.C2GM.DEF.LNE.FEBA": {
                   "symbolID": "G*GPDLF---****X",
                   "tag": "FORWARD EDGE OF BATTLE AREA",
-                  "symbolType": "MultiSegmentLine"
+                  "symbolType": "SimpleLine"
```

Comment:
By making this a `SimpleLine`, we only ever get the two X-Men symbols on either end ... desirable?